.github/workflows/test.yml

```diff
@@ -65,10 +65,19 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose --workspace
+    - run: apt-get update && apt-get install jq -y
+    - name: build integration-test binary for subxt
+      run : eval "cp $(cargo test --no-run --message-format=json | jq '. | select(.executable != null) | .executable') test-binary"
+      working-directory: ./integration/subxt-tests
```

Comment:
Because it'll need substrate to be running in the background, which needs docker image. Which solang CI image don't provide. So I build the test binary and run it later.

Reply:
The substrate tests don't run in a container (let alone the Solang CI container). These tests run the CI VM and both `docker`  and `cargo` should be available in this environment

Reply:
I've clean this up so it's not carried over anymore, currently not running due to tests failing after rebase.